src/mixins/filing-template-mixin.ts

```diff
@@ -87,6 +97,11 @@ export default class FilingTemplateMixin extends Vue {
         valid: false,
         certifiedBy: draftFiling.header.certifiedBy
       })
+
+      // Set Future Effective Time
+      this.setFutureEffectiveDate(draftFiling.header.effectiveDate)
+      this.setIsImmediate(!draftFiling.header.effectiveDate)
+      this.setIsFutureEffective(!!draftFiling.header.effectiveDate)
```

Comment:
Why do we need 2 properties, since they are (always?) opposite of each other?

Reply:
I see you use this for the radio buttons, but the fact that both radio buttons are initialized as "not set" should be abstracted in the component. At the app level, we only need to know if immediate or future, right?

Reply:
Yes for sure, let me look into this a bit deeper. If we can eliminate the isImmediate prop, will do so! Good idea, will get back to this asap.

Reply:
Removed the `isImmediate` property outside its local use to the component 👍